docs/ml-guide.md

```diff
@@ -192,6 +192,16 @@ Parameters belong to specific instances of `Estimator`s and `Transformer`s.
 For example, if we have two `LogisticRegression` instances `lr1` and `lr2`, then we can build a `ParamMap` with both `maxIter` parameters specified: `ParamMap(lr1.maxIter -> 10, lr2.maxIter -> 20)`.
 This is useful if there are two algorithms with the `maxIter` parameter in a `Pipeline`.
 
+## Saving and Loading Pipelines
+
+Often times it is worth it to save a model or a pipeline to disk for later use. In Spark 1.6, a model import/export functionality was added to the Pipeline API. Most basic transformers are supported as well as some of the more basic ML models such as:
+
+* K-Means
+* Naive Bayes
+* ALS
+* Linear Regression
+* Logistic Regression
+
```

Comment:
Could you comment here telling users to refer to the algorithms' API documentation to see if the algorithm support save/load?